app/assets/stylesheets/welcome.scss

```diff
@@ -2,3 +2,40 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 $backdrop : #C9D6AE;
+
+.welcome {
+  background-image: image-url('skyscrapers.jpg');
+  background-repeat: no-repeat;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  min-height: 100%;
+  min-width: 1024px;
+  }
+
+.top-bar {
+  position-left: 0;
+  position-right: 0;
+  width: 100%;
+}
+
+.fullWidth {
+   width: 100%;
+   margin-left: auto;
+   margin-right: auto;
+   max-width: initial;
+}
+
+#welcome-title {
+  font-size: 80px;
+  color: white;
+  margin-left: 10px;
+  word-wrap: break-word;
+}
+
+#welcome-description {
+  font-size: 24px;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.